lte/gateway/c/core/oai/CMakeLists.txt

```diff
@@ -41,9 +41,9 @@ add_definitions(-DCMAKER)
 # Generate *.gcno rather than *.c.gcno files for gcov
 set(CMAKE_C_OUTPUT_EXTENSION_REPLACE 1)
 
-set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS} -fstack-protector-all -g -DMALLOC_CHECK_=3 -DDEBUG_IS_ON=1 -DTRACE_IS_ON=1 -O0 -fsanitize=address -fsanitize=undefined -fno-omit-frame-pointer")
```

Comment:
Why do we have to disable ASAN/LSAN for this benchmarking?

Reply:
+1, same question regarding ASAN

Reply:
Forgot to re-enable it after valgrind trials